src/client/Microsoft.Identity.Client/Platforms/netstandard13/NetStandard13DeviceAuthManager.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Identity.Client.PlatformsCommon.Interfaces;
+
+namespace Microsoft.Identity.Client.Platforms.netstandard13
+{
+    internal class NetStandard13DeviceAuthManager : IDeviceAuthManager
```

Comment:
Why are there 2 of these? netstandard13 and netstnadard?

Reply:
I remaned this but the second file stuck around for some reason. will be removed